### PR TITLE
YALB-768: local development scripts

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
## [YALB-768: local development scripts](https://yaleits.atlassian.net/browse/YALB-768)

**Description of work**
- Updates all of the local scripts to be useful, and work as expected. Follow review steps below for details.

### Setup the project from scratch
- [ ] `lando stop` Kill any running lando instance
- [ ] `docker system prune -a` delete all currently unused docker images and stuff (next time you start, it'll download updated/latest ones)
- [ ] Quit, and restart Docker, just for posterity.
- [ ] Delete your existing `yalesites-project` folder... or rename it or something
- [ ] Clone the repo `git clone git@github.com:yalesites-org/yalesites-project.git`
- [ ] `cd yalesites-project`
- [ ] `npm run setup`
	- Go find something else to do... This will take a LONG time, since it's cloning all of the docker containers from scratch, and doing a full install of the project from scratch. You might think it's frozen a couple of times. IT'S NOT. It just takes a REALLY long time!)
	- You will be asked for your ssh password about mid-way through, so check back occasionally
- [ ] Click the login link, and view your awesome site!

_NOTE: You can re-run the `npm run setup` script later to get a fresh install, and it'll run much faster since you'll already have the docker images cached locally. I just went a little extra-overboard in the steps above to verify new contributors would be successful setting the project up for the first time._

### CODE REVIEW

File: scripts/local/db-get.sh
Comment (nJim) [inline scripts/local/db-get.sh:4]: I'm confused by this bit. Seems like you want to keep the old version as a backup so you do not lose it. What's the reason? If you run this command twice then you overwrite it anyhow.

Comment (ModulesUnraveled) [reply scripts/local/db-get.sh:4]: Yeah, it's really just a light weight "oh crap I wanted to keep using that other db, but I just ran the db:get script, and it's gone now" safeguard. But I didn't think it was necessary to keep anything more than just the previous one. I guess it's not necessary at all, you could always download it directly from pantheon manually. It's something I've used before, but I'm not tied to in any way.

Comment (ModulesUnraveled) [reply scripts/local/db-get.sh:4]: It would probably be most useful if there was a major database change, but you are already working on a PR before that update happened. This way you could have both database states locally for different branches and import the one you need.

It's definitely an edge case though, so if you're proposing getting rid of this, I wouldn't fight it.

File: package.json
Comment (nJim) [inline package.json:23]: I'm not able to figure out this line. I think you are already doing it, but we should describe some of these in a README file.

**Review by nJim — COMMENTED** [review]
This is a big step in the right direction. Thank you for cleaning up so much of this. I left a few questions. Feel free to use the feedback in your next round of edits .. I can mark this approved if you want to just move it forward as there is no risk to the application.

File: scripts/local/db-get.sh
Comment (nJim) [reply scripts/local/db-get.sh:4]: Let's keep it for now. It doesn't hurt anything and offers a small assurance. But I may change this when we revamp local script in the future.

File: scripts/local/assets-pull.sh
Comment (vinmassaro) [inline scripts/local/assets-pull.sh:3]: Why does this pull from test?

File: scripts/local/local-review-with-atomic-branch.sh
Comment (vinmassaro) [reply scripts/local/local-review-with-atomic-branch.sh:9]: There are a few quick [shellcheck](https://www.shellcheck.net/) fixes that can be made on this file

File: scripts/local/local-review-with-cl-branch.sh
Comment (vinmassaro) [inline scripts/local/local-review-with-cl-branch.sh:16]: There are a few quick [shellcheck](https://www.shellcheck.net/) fixes that can be made on this file

File: scripts/local/local-review-with-atomic-branch.sh
Comment (ModulesUnraveled) [reply scripts/local/local-review-with-atomic-branch.sh:9]: I fixed this in the follow-up PR Here: https://github.com/yalesites-org/yalesites-project/pull/147

**Review by vinmassaro — CHANGES_REQUESTED** [review]
I added a few minor comments, but other than that, this looks good to me. Thanks for doing this cleanup!

File: scripts/local/assets-pull.sh
Comment (ModulesUnraveled) [reply scripts/local/assets-pull.sh:3]: It shouldn't. I didn't look closely enough. I've updated it in the follow-up PR to pull from dev.

File: scripts/local/local-review-with-cl-branch.sh
Comment (ModulesUnraveled) [reply scripts/local/local-review-with-cl-branch.sh:16]: I've made all of these fixes in the [followup PR](https://github.com/yalesites-org/yalesites-project/pull/147) Thanks!

**Review by nJim — APPROVED** [review]
I see the changes in the follow up PR. Thanks Brian. Approved from my point of view. Will also wait for Vincent to sign off to call it ready to merge.

**Review by vinmassaro — APPROVED** [review]

**Pull request merged**
